### ISSUE TITLE
"Unused open statements" analyzer does not work on a very simple code

### ISSUE THREAD

**Issue by vasily-kirichenko** [Microsoft/visualfsharp#3775]
![image](https://user-images.githubusercontent.com/873919/31741000-a1d090ea-b45b-11e7-842a-469501f7418e.png)

```fsharp
open System
open System.Threading

[<EntryPoint>]
let main _ =
    printfn "%A" System.DateTime.Now
    0
```

**Comment by EBrown8534** [comment]
Shouldn't this emit that the name of `System.DateTime.Now` can be simplified instead?

**Comment by vasily-kirichenko** [comment]
no, that’s the work of a different analyzer.

**Comment by vasily-kirichenko** [comment]
this, and many other bugs, has been fixed in https://github.com/Microsoft/visualfsharp/pull/3803

**Comment by EBrown8534** [comment]
@vasily-kirichenko I meant that this analyzer shouldn't run, and the other one _should_. I would think that if you `open` something, and call a method with the FQN including that `open`ed thing, it would not emit this suggestion but instead allow the other analyzer to emit a "name can be simplified" suggestion.

**Comment by vasily-kirichenko** [comment]
1. turn on the simplify name analyzer in the settings 2. build and install the vsix from my pr I mentioned. After that _everything_ will work properly.

**Comment by cartermp** [comment]
@EBrown8534 The Simplify Name analyzer is off by default because it's expensive and bogs down the editor due to how document analyzers are run by Roslyn under the covers.  You can see a breakdown of what happens in the editor here: https://github.com/dotnet/roslyn/issues/19347#issuecomment-310225540

**Comment by EBrown8534** [comment]
@cartermp Ah, thanks! That clears things up for me.

**Comment by cartermp** [comment]
Fixed by #3803
